recipes/mne-python_1.4/construct.yaml

```diff
@@ -17,11 +17,12 @@ conclusion_file: ../../assets/conclusion.rtf  # [not win]
 initialize_by_default: False
 register_python_default: False
 
-# default_prefix will be ignored by macOS .pkg installer!
 default_prefix: ${HOME}/mne-python/1.4.2_0                          # [linux]
 default_prefix: "%USERPROFILE%\\mne-python\\1.4.2_0"                # [win]
 default_prefix_domain_user: "%LOCALAPPDATA%\\mne-python\\1.4.2_0"   # [win]
 default_prefix_all_users: "%ALLUSERSPROFILE%\\mne-python\\1.4.2_0"  # [win]
+default_location_pkg: "/Applications"  # [osx]
```

Comment:
This should not contain the `/`.

Also note the shortcut will _also_ be created in Applications, so it might be confusing to have two things named like that. I usually put the full installation in `~/Library` and then let the shortcut live in `~/Applications`.

Reply:
Ahh then the example in https://github.com/conda/constructor/blob/main/CONSTRUCT.md should be updated, it mentions using `/Library` so I guess it should be `Library`?